packages/access-api/src/ucanto/proxy.js

```diff
@@ -28,18 +74,24 @@ export function createProxyHandler(options) {
       : // this works, but involves lying about the issuer type (it wants a Signer but context.id is only a Verifier)
         // @todo obviate this type override via https://github.com/web3-storage/ucanto/issues/195
         /** @type {Ucanto.Signer} */ (context.id)
-
-    const [result] = await Client.execute(
-      [
-        Client.invoke({
-          issuer: proxyInvocationIssuer,
-          capability: capabilities[0],
-          audience,
-          proofs: [invocationIn],
-        }),
-      ],
-      /** @type {Client.ConnectionView<any>} */ (connection)
-    )
-    return result
+    const proxyInvocation = Client.invoke({
+      issuer: proxyInvocationIssuer,
+      capability: capabilities[0],
+      audience,
+      proofs: [invocationIn],
```

Comment:
github is forcing me to add another comments because i approved instead of requesting changes lol

Reply:
1. Why? I don't think that will add anything. All that info will already be in the resulting `proxyInvocation` proof chain because the whole `invocationIn` (including its `expiration` is in `proofs`). wdyt @Gozala 
2. That's not related to the point of this PR, so if you're proposing changing the proxy logic, let's do it as a separate issue.

Reply:
This also prompted me to look at take [another look at the spec](https://github.com/ucan-wg/spec/#61-time) and it seems to state

> All proofs MUST contain time bounds equal to or broader than the UCAN being delegated. If the proof expires before the outer UCAN — or starts after it — the reader MUST treat the UCAN as invalid.

Which is to say that UCANs that not copying those time bounds would render a UCAN invalid, that said I'm not sure if this is how ucanto does it, but I'll check

Reply:
Created https://github.com/web3-storage/ucanto/issues/198

Reply:
> Which is to say that UCANs that not copying those time bounds would render a UCAN invalid, that said I'm not sure if this is how ucanto does it, but I'll check

I read it differently... in this case of proxying, the incoming `invocationIn` becomes the proof. Let's say it expires in 24hrs. Let's say without copying over that expiry, the `proxyInvocation` gets created with a 30sec expiry.

> All proofs MUST contain time bounds equal to or broader than the UCAN being delegated.

The time bounds of the proof `invocationIn` is [t0, 0 + 24hrs]. The time bounds of 'the UCAN being delegated' I think is the `proxyInvocation` whose time bounds is [t1, t1 + 30sec].  Assuming `t1 - t0 ~= 0`, then I think it's fair to say the time bounds of the proof are broader than the invocation/delegation. 

>  If the proof expires before the outer UCAN — or starts after it — the reader MUST treat the UCAN as invalid.

The proof would not expire before the 'outer UCAN'.

Reply:
But it would be bad if `invocationIn` expiry was like 15sec, and the `proxyInvocation` expiry defaulted to 30sec. So that makes it worth copying the expiration for that scenario